### PR TITLE
Submit “estimation-of-COVID-19-pandemic.ipynb”

### PR DESCRIPTION
**Pull request by WHQWHQWHQ**
… the test of None, empty, and invalid situation. And add ornament markdown before each test cases

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [x] review the artifacts from local build
- [ ] [local build the slides](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [x] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have applied tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb [hunk 1]
```diff
@@ -119,19 +119,32 @@
    "outputs": [],
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #15.        return df[df[column_name] == column_value]
</pre><p>Please just revert the changes in this cell. You don't need to add such more checks. They are not necessary. Also, you don't need to check <code>if column_value not in df[column_name].values:</code></p><p> </p><p>But all your new added test cases are very good. You can keep them, but you may need to fix the assert a little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #6.        if not isinstance(column_name, str) or column_name is None or not column_name.strip():
</pre><p>remove this check </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #4.        if year is None or not isinstance(year, int) or year &lt; 0:
</pre><p>Good check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #4.        if column is None or not isinstance(column, str) or column not in df.columns or not column.strip():
</pre><p>check <code>column not in df.columns&nbsp;</code> is enough </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #4.        if countries_and_region is None or not isinstance(countries_and_region, str) or countries_and_region.strip() == "":
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #4.        if column_name is None or not isinstance(column_name, str) or column_name not in df.columns or not column_name.strip():
</pre><p>same as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <pre class="ql-syntax" spellcheck="false">Line #4.        if column_name is None or not isinstance(column_name, str) or column_name not in df.columns or not column_name.strip():
</pre><p>same as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p>

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb:119]: <p>Done</p><p> </p>

**Pull request merged**